SchedulePlatform.Models/Entities/Customer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace SchedulePlatform.Models.Entities
+{
+    public class Customer: BaseEntity
+    {
+        public string? FirstName { get; set; }
+        public string? LastName { get; set; }
```

Comment:
Consistency again. The Appointment entity has a empty new line between its properties but this one doesn't. Please choose one option and use it evrywhere.